introduction.md

```diff
@@ -9,3 +9,4 @@ My deadly skill are:
 * Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?